cmd/notifyPendingCI.go

```diff
@@ -86,9 +85,12 @@ var notifyPendingCICmd = &cobra.Command{
 				}
 
 				if shouldSendMessage {
-					message := fmt.Sprintf("The following pull request is either waiting for approval for CI checks to run or has failed checks: %s", pr.URL)
+					status := "firing"
+					title := "The following pull request is either waiting for approval for CI checks to run or has failed checks:"
+					description := pr.URL
```

Comment:
I thought URL is usually the API's URL, and the HTML URL element is actually what a human reader would expect to get?